src/beaker/sqlwarehouseutils.py

```diff
@@ -135,6 +138,27 @@ def launch_warehouse(self, config):
                 "Allowed types include: ['warehouse', 'cluster']."
             )
 
+        # Determine the Warehouse type (classic, pro, serverless) to launch
+        # warehouse_type: PRO or CLASSIC. If you want to use serverless compute, you must set to PRO and also set the field enable_serverless_compute to true.
+        enable_serverless_compute = True
+        if type == "warehouse":
+            # default to serverless
+            if "warehouse" not in config or config["warehouse"] == "serverless": 
+                warehouse_type = "PRO"
```

Comment:
Should this be "PRO" or "SERVERLESS"?

Reply:
the API allows creation of serverless compute with 2 parameters:
"Warehouse type: PRO or CLASSIC. If you want to use serverless compute, you must set to PRO and also set the field enable_serverless_compute to true."

https://docs.databricks.com/api/workspace/warehouses/create